java/gradle-plugin/.idea/modules/bond-gradle.iml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<module external.linked.project.id="bond-gradle" external.linked.project.path="$MODULE_DIR$/../.." external.root.project.path="$MODULE_DIR$/../.." external.system.id="GRADLE" external.system.module.group="com.microsoft.bond" external.system.module.version="1.0" type="JAVA_MODULE" version="4">
+<module external.linked.project.id="bond-gradle" external.linked.project.path="$MODULE_DIR$/../.." external.root.project.path="$MODULE_DIR$/../.." external.system.id="GRADLE" external.system.module.group="org.bondlib" external.system.module.version="1.0" type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager">
-    <output url="file://$MODULE_DIR$/../../build/classes/main" />
-    <output-test url="file://$MODULE_DIR$/../../build/classes/test" />
+    <output url="file://$MODULE_DIR$/../../out/production/classes" />
```

Comment:
Wasn't originally, but I'm consciously surrendering and making that change here:
* I have yet to find a way to get IDEA to stop changing it back.
* I don't know how to get IDEA to even temporarily "compile" core's `alphabet.txt` test resource into `build/` instead of `out/`, so there is going to be a `core/out/` either way.
* `java/.gitignore` includes `*/out` as of this PR.

There wasn't a strong reason for trying to share a build directory in the first place, and some internet folks seem to think it's actually a [bad idea](https://youtrack.jetbrains.com/issue/IDEA-91952).